### ISSUE TITLE
Add XDG Base Directory support

### ISSUE THREAD

**Issue by akiva** [5digits/dactyl#127]
It would be great to see this implemented, allowing those systems that use the spec to use `~/.config/pentadactyl/`, falling back to `~/.pentadactyl/` for those that do not. 

Further, storing the config file within this directory, (ie. `~/.config/pentadactyl/pentadactylrc`), would be highly desirable.

**Comment by jakejohns** [comment]
I believe the following will XDG-ify it:
```bash
export PENTADACTYL_RUNTIME="$XDG_CONFIG_HOME/pentadactyl"
export PENTADACTYL_INIT=":source $PENTADACTYL_RUNTIME/pentadactylrc"
```

**Comment by akiva** [comment]
Yes, it would, and I do things like this for programs that do not support the XDG standard. However, I am requesting built-in support as it's very easy to do. If I have some time in the next week, I can even do a PR to add this.

**Comment by jakejohns** [comment]
fair enough, @akiva. I think default support is probably better as well.
